INSTALL.md

```diff
@@ -89,63 +110,107 @@ To build **pokecrystal.gbc**:
 make
 ```
 
+To build **pokecrystal11.gbc**:
+
+```bash
+make crystal11
+```
+
 
 ## Linux
 
-### Install the required software
+Open **Terminal** and enter the following commands.
 
-#### Debian/Ubuntu
+### 1. Install the required software
+
+The commands to do this depend on which distro you're using.
+
+#### Debian or Ubuntu
+
+To install the software required for **pokecrystal**:
 
 ```bash
-sudo apt-get install make gcc bison git libpng-dev
+sudo apt-get install make gcc git
+```
 
+To install **rgbds**:
```

Comment:
Technically, make gcc and git are also required to build rgbds... I don't really see the point in splitting this off.

Reply:
To make it clear which packages are only required for building rgbds, and so the `git clone` and `sudo make` commands common to all distros are more obvious. (I was considering a final section that just has the `git clone`, `cd pokecrystal`, `make pokecrystal`, `make pokecrystal11` commands, and every OS section would link to it; does that sound like an improvement?)

Reply:
Yeah, that does sound like an improvement, with proper links.
Just say "continue by [installing pokecrystal]".